src/controls/trackpad-controls.js

```diff
@@ -59,24 +60,63 @@ module.exports = AFRAME.registerComponent('trackpad-controls', {
   },
 
   onTouchStart: function (e) {
-    this.isMoving = true;
+    this.startingAxisData = [];
+    this.canRecordAxis = true
     e.preventDefault();
   },
 
   onTouchEnd: function (e) {
+    this.startingAxisData = []
     this.isMoving = false;
     e.preventDefault();
   },
 
   onAxisMove: function(e){
     var axis_data = e.detail.axis;
 
-    if(axis_data[1] < 0){
-      this.zVel = 1;
+    if(this.canRecordAxis){
```

Comment:
Instead of having a `canRecordAxis` property, how about just checking `startingAxisData.length === 0`?